src/common/Button/Button.types.ts

```diff
@@ -1,5 +1,5 @@
 export interface ButtonProps {
-	id: string;
+	id?: string;
 	text: string;
-	onClick: () => void;
+	onClick?: () => void;
```

Comment:
do you really need a button without onClick prop?) I think it the main reason why we use buttons) to click on them